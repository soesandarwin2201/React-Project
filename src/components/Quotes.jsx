/*eslint-disable*/
import React, { PureComponent } from 'react';

class Quotes extends PureComponent {
  render() {
    return (
      <div className='quote'>
        <blockquote>
          Mathematics is not about numbers, equations, comutation or algorithms:
          It is about understanding{' '}
        </blockquote>
        <cite>WIlliam Paul Thurston</cite>
      </div>
    );
  }
}

export default Quotes;
