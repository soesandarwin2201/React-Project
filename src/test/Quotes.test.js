import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../components/Quotes';

it('render correctly', () => {
  const tree = renderer.create(<Quotes />).toJSON();
  expect(tree).toMatchSnapshot();
});
