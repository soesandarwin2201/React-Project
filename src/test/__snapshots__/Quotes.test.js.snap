// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div
  className="quote"
>
  <blockquote>
    Mathematics is not about numbers, equations, comutation or algorithms: It is about understanding 
  </blockquote>
  <cite>
    WIlliam Paul Thurston
  </cite>
</div>
`;
